RedHatInsights/compliance-frontend
# feat(ESSNTL-3718): Add Inventory group filter and column

Implements https://issues.redhat.com/browse/ESSNTL-3718.

Column: "Group", non-sortable, hidden if the unleash feature flag is set to false.
Filter: multiple-choice, renders chips, hidden if the unleash feature flag is set to false, mustn't update URL.

- [x] /systems: support column
- [x] /systems: support filter
- [x] /reports/%id: support column
- [x] /reports/%id: support filter
- [x] /scappolicies/new: support column
- [x] /scappolicies/new: support filter

## How to test

In the views that are updated with the new filter, try to filter out hosts by groups. You can create groups/add new hosts beforehand. Check that the hosts have correct group name in the Group column.

## Screenshots

/systems
![image](https://github.com/RedHatInsights/compliance-frontend/assets/31385370/a5b0927d-7e43-43ac-931b-5778a0181c47)

/scappolicies/new
![image](https://github.com/RedHatInsights/compliance-frontend/assets/31385370/0d2a0d74-ebe7-4ec7-bf76-17a7e8a585fc)

/reports/%id
![image](https://github.com/RedHatInsights/compliance-frontend/assets/31385370/3df4bc3a-5f79-4859-8494-d1e1db3c49c6)
